radicle-httpd/src/api/v1/projects.rs

```diff
@@ -1310,4 +1310,34 @@ mod routes {
             )
         );
     }
+
+    #[tokio::test]
+    async fn test_blob_stream() {
+        use crate::api::json::blob_stream;
+        use futures_util::StreamExt;
+
+        let tmp_dir = tempfile::tempdir().unwrap();
+        std::fs::create_dir_all(tmp_dir.path()).unwrap();
+        let tmp_file = tmp_dir.path().join("test_blob_stream");
+
+        let blob = "This is a test blob"; // 19 chars.
```

Comment:
is comment needed?

Reply:
It's helpful for me to read my own test code ;-)